source/agora/consensus/data/Block.d

```diff
@@ -449,15 +450,16 @@ public Block makeNewBlock (Transactions)(const ref Block prev_block,
     block.txs.sort;
 
     block.header.merkle_root = block.buildMerkleTree();
+    block.header.enrollments = enrollments;
```

Comment:
Hum they aren't sorted ??

Reply:
Added sort logic.

Reply:
The part I was worried about is whether I should align here once more.

Reply:
I thought it'd be better to check if it's aligned here.